src/planners/path_search.jl

```diff
@@ -47,3 +47,26 @@ Base.iterate(sol::PathSearchSolution) = iterate(sol.plan)
 Base.iterate(sol::PathSearchSolution, istate) = iterate(sol.plan, istate)
 Base.getindex(sol::PathSearchSolution, i::Int) = getindex(sol.plan, i)
 Base.length(sol::PathSearchSolution) = length(sol.plan)
+
+
+"Solution type for search-based planners that produce fully ordered plans."
+mutable struct BiPathSearchSolution{S<:State,T} <: OrderedSolution
+    status::Symbol
+    plan::Vector{Term}
+    trajectory::Union{Vector{S},Nothing}
```

Comment:
Maybe remove `trajectory` from the search solution, as noted above.

Reply:
On second thought, maybe the partial forward trajectory of complete states should be stored as `f_trajectory`, and the backward trajectory of partial states should be stored as `b_trajectory`. Storing this information would help us check the correctness of the frontier crossing code more easily using a unit test.